src/Orchard.ResourceManagement/TagHelpers/ScriptTagHelper.cs

```diff
@@ -134,6 +173,31 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 {
                     definition.SetVersion(Version);
                 }
+                
+                // If At is specified then we also render it
+                if (At != ResourceLocation.Unspecified)
+                {
+                    var setting = _resourceManager.RegisterResource("script", Name);
+
+                    setting.AtLocation(At);
+
+                    if (!String.IsNullOrEmpty(Condition))
+                    {
+                        setting.UseCondition(Condition);
+                    }
+
+                    setting.UseDebugMode(Debug);
```

Comment:
It's already the case I think, there is a setting in site settings, and it also take the environment into account. I abstracted the logic in ResourceManagement.